### PR TITLE
HKG CAN FD HDA1: show FCA warning light

### LINKED ISSUE #1157: ev6 w/o HDAII doesn't display FCA/AEB warning when in OP long
### Describe the bug

It's my understanding (based on the warning message about OP long being alpha) that on an ev6 with HDA I, enabling OP longitudinal control will disable stock AEB and FCA. The indicator on the dashboard for my ev6 definitely does not have any warning lights illuminated, however. I don't know if this means that stock AEB/FCA is still enabled, or if it is disabled but the warning lights aren't properly illuminating.

### Which car does this affect?

Kia EV6 2023 Wind AWD (HDA I)

### Provide a route where the issue occurs

fab807792f63fea6|2023-09-15--18-57-23--0

### openpilot version

master-ci

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by sshane**
Fixes: https://github.com/commaai/opendbc/issues/1157

Even if AEB still works since the camera does it, we still want to show since we haven't verified that fact (same with Ford and Bolt).

- [ ] do safety

### CODE REVIEW

File: selfdrive/car/hyundai/carcontroller.py [hunk 1]
```diff
@@ -126,6 +126,9 @@ def update(self, CC, CS, now_nanos):
       if self.CP.openpilotLongitudinalControl:
         if hda2:
           can_sends.extend(hyundaicanfd.create_adrv_messages(self.packer, self.CAN, self.frame))
+        else:
+          # 0x160 is sent by camera
```
Comment (sshane) [inline selfdrive/car/hyundai/carcontroller.py:130]:
```suggestion
```

**Comment by bdimcheff** [comment]
seems to have worked! aeb warning light is on when OP long is on, and off in chill mode/stock long. I didn't actually try to drive anywhere.

I'm happy to try to figure out of aeb is still on anyways if you have a suggestion besides trying to crash the car ;)

File: selfdrive/car/hyundai/hyundaicanfd.py [hunk 2]
```diff
@@ -191,6 +183,20 @@ def create_adrv_messages(packer, CAN, frame):
       'SET_ME_9': 0x9,
     }
     ret.append(packer.make_can_msg("ADRV_0x160", CAN.ECAN, values))
+  return ret
+
+
+def create_adrv_messages(packer, CAN, frame):
+  # messages needed to car happy after disabling
```
Comment (bdimcheff) [inline selfdrive/car/hyundai/hyundaicanfd.py:190]: I realize this is just a moved comment but "needed to car happy" is missing a verb or something.

**Comment by sshane** [comment]
@bdimcheff Try it out, engaging and everything? There's one static byte in the message that's different from our EV6, so I'd like to figure out what causes that (if it's a user setting or constant). Can you try to get another route where you toggle all features related to ADAS/FCA/FCW on and off a few times and in different combinations?

**Comment by bdimcheff** [comment]
@sshane `fab807792f63fea6|2023-09-25--16-45-05--0` I started out with all safety features on. Changed follow distance through the full cycle. Switched from AEB -> warning only mode and engaged/disengaged again. Then, all forward collision features off and toggled on and off. Then I turned side-collision into warning only mode and then off. I don't think there's any other toggle I could toggle at that point.

**Comment by sunnyhaibin** [comment]
@sshane What else is needed for this PR? Multiple users have been testing this on sunnypilot and have been working with no issues.

**Comment by sshane** [comment]
We've moved the car interfacing code to our [`opendbc`](https://github.com/commaai/opendbc) repository, which is now the new home for car ports and fingerprints. Please re-open your pull request against [`opendbc`](https://github.com/commaai/opendbc) at your convenience by using this command below. This will transform all changes under `selfdrive/car/` to `opendbc_repo/opendbc/car/`. Make sure you have initialized submodules beforehand and have checked out the latest `opendbc` commit.

```bash
PR_NUMBER=33045
curl -L https://github.com/commaai/openpilot/pull/$PR_NUMBER.patch | sed -e 's/selfdrive\/car/opendbc_repo\/opendbc\/car/g' | git apply -v --reject
```

Simply replace the PR number with your own. Once done, add the files, fix any conflicts, and open a new PR. Alternatively, you may start a new PR from scratch if that is easier for you.

**Pull request closed without merging**